packages/cozy-client/src/CozyClient.js

```diff
@@ -136,7 +136,10 @@ export default class CozyClient {
 
   async destroy(document, mutationOptions = {}) {
     await this.triggerHook('before:destroy', document)
-    const res = await this.mutate(Mutations.deleteDocument(document), mutationOptions)
+    const res = await this.mutate(
+      Mutations.deleteDocument(document),
+      mutationOptions
+    )
     await this.triggerHook('after:destroy', document)
```

Comment:
Hmmm I don't see how to avoid it, hooks must be executed in that order (I don't like this hook impl, but that's another problem). Do you have any idea?

Reply:
Unfortunately I don't have any ideas to propose, I saw this code in the PR, but I don't know what this code does, or how the code is structured, so it's hard to propose a solution to avoid this side effect, which is still a smell difficult to maintain.